coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -22,11 +22,13 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
         """
 
         if len(result.affected_code) == 0:
-            return False
+            return 'There is no code to be ignored'
```

Comment:
@sils IMO result should not be displayed in front of users as he doesn't know what result refers to.